src/config/dark-sites.config

```diff
@@ -169,6 +167,12 @@ heavybit.com
 hidive.com
 himovies.to
 hostedtalk.net
+humblebundle.com/$
```

Comment:
Why doing this well already refericing to other sub-url's?

Reply:
Because those other sub-url's have dark mode.